swf/src/types/rectangle.rs

```diff
@@ -0,0 +1,15 @@
+/// A rectangular region defined by minimum and maximum x- and y-coordinate positions.
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
```

Comment:
This should probably derive `Copy` too.

Reply:
I removed `Copy` from `Rectangle` in fb39bb2b713ac0385ebd8f8f43ecc87b78e163b0, in order to make clones explicit.